app_feup/lib/view/Pages/LoginPageView.dart

```diff
@@ -4,30 +4,44 @@ import 'package:flutter_redux/flutter_redux.dart';
 import 'package:flutter_svg/svg.dart';
 import 'package:toast/toast.dart';
 import '../../view/Theme.dart';
-import 'dart:async';
-
-class LoginPageView extends StatelessWidget {
-  LoginPageView(
-      {Key key,
-      @required this.checkboxValue,
-      @required this.setCheckboxValue,
-      @required this.usernameFocus,
-      @required this.passwordFocus,
-      @required this.usernameController,
-      @required this.passwordController,
-      @required this.formKey,
-      @required this.submitForm})
-      : super(key: key);
-
-  final bool checkboxValue;
-  final Function setCheckboxValue;
-  final Function submitForm;
-  final FocusNode usernameFocus;
-  final FocusNode passwordFocus;
-  final TextEditingController usernameController;
-  final TextEditingController passwordController;
-  final GlobalKey<FormState> formKey;
+import 'package:app_feup/redux/ActionCreators.dart';
+
+
+class LoginPageView extends StatefulWidget {
+  @override
+  _LoginPageViewState createState() => _LoginPageViewState();
+}
+
+class _LoginPageViewState extends State<LoginPageView> {
+  final String faculty = 'feup';
+
+
+  static final FocusNode usernameFocus = FocusNode();
+  static final FocusNode passwordFocus = FocusNode();
+
+  static final TextEditingController usernameController = TextEditingController();
+  static final TextEditingController passwordController = TextEditingController();
```

Comment:
Username and password controllers were made static final, so when the user logs out the username and password fields will still have their previous content.

Reply:
Thanks for noticing this!